src/app/static/static-values.ts

```diff
@@ -62,10 +62,6 @@ export class StaticValues {
     'SUSHI_MIC_USDT',
     'SUSHI_MIS_USDT',
     'CRV_OBTC',
-    'ONEINCH_ETH_DAI',
-    'ONEINCH_ETH_USDC',
```

Comment:
in this list, we have all strategies
we can have withdrawals from an old one

Reply:
reverted